articles/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-active-directory-b2c-oidc.md

```diff
@@ -49,7 +49,7 @@ In this tutorial, you learn how to:
    ![Fill in the values to generate the project](media/configure-spring-boot-starter-java-app-with-azure-active-directory-b2c-oidc/fill-in-the-values-to-generate-the-project.png)
 
     > [!NOTE]
-    > Spring Initializr uses Java 11 as the default version. To use the Spring Boot Starters described in this topic, you must select Java 8 instead.
```

Comment:
We could remove this [!NOTE] section since we are not noticing users of selecting Java 8.